aws/cloudformation/components/database.yml.erb

```
@@ -138,6 +138,7 @@
       PermissionsBoundary: !ImportValue IAM-DevPermissions
   ReportingDBProxy:
     Type: AWS::RDS::DBProxy
+    DependsOn: [ReaderSQLUser]
```

```suggestion
    DependsOn: ReaderSQLUser
```

The DependsOn attribute can take a single string or list of strings.